src/dhtproto/client/UsageExamples.d

```diff
@@ -162,6 +162,9 @@ version ( UnitTest )
                     this.log.error(this.msg_buf);
                     break;
 
+                case none:
```

Comment:
Hmm, I feel like this should be an `assert(false)` as well... There's no way a null notification should ever happen. I wonder if this could be combined into the proposed mixin (in `ocean.core.SmartUnion`?).

Reply:
How is this comment outdated I have no idea. Yes, my intention was to write `assert(false);` here.